.github/workflows/daily.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  daily:
+    runs-on: ubuntu-latest
+    if: contains(github.ref, 'refs/heads/master')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Generate top 100
+        run: go run . -count 100 > top-100.tsv
+
+      - name: Generate top 1000
+        run: go run . -count 1000 > top-1000.tsv
+
+      - name: Generate top 10000
+        run: go run . -count 10000 > top-10000.tsv
+
+      - name: Daily update
+        uses: drud/action-cross-commit@master
```

Comment:
please pin a specific full commit here. I don't want to give another user the ability to do anything with my github token by just pushing to that repository's master branch and waiting a few days.

Reply:
Pinned on [`13266270ee2da98f1d16425f6ce21c235a7f33dd`](https://github.com/drud/action-cross-commit/commits/13266270ee2da98f1d16425f6ce21c235a7f33dd)